website/templates/profile/account.mako

```diff
@@ -66,12 +65,15 @@
                                 <tr>
                                     <td colspan="3">
                                         <form data-bind="submit: addEmail">
+                                        <p>To merge an existing OSF account with this one or to be able to log in under multiple email addresses, add your alternate email address in the field below.</p>
```

Comment:
@abought Fair enough; I think having the text is fine if it adds clarity, which it does.

Still wouldn't hurt to have @erinmayhood to do a quick review of the text, as the discussion on the ticket happened after the product demo stage.